lib/prompts/installPublicAppPrompt.ts

```diff
@@ -47,6 +46,4 @@ const installPublicAppPrompt = async (
     `&redirect_uri=${encodeURIComponent(redirectUrls[0])}`;
 
   open(url);
-};
-
-module.exports = { installPublicAppPrompt };
+}
```

Comment:
type `promptUser`

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1288/commits/dbfad7214b180917cc226ffaee59f96b198bf5f3